andela/tesla-ah-frontend
# #166240967 Add search functionality to the project

### What does this PR do?
> Add search functionality to Authors Haven

### Description of a task to be completed?
A user will be able to search an article by:
- Author
- Title
- Tags
- Keywords

### How should it be manually tested?
- Clone this repo and `cd` into it.
- Run `yarn` or `yarn install` to install and refresh packages
- Run `yarn dev` to run the project
- Click the search icon which redirects you to the search page
- Type in the search box and choose a filter.
- Click the search submit button, then you see the search results.

### Pivotal tracker story
- [#166240967](https://www.pivotaltracker.com/story/show/166240967)

### Screenshot
![search](https://user-images.githubusercontent.com/22316636/63494338-c08f7180-c4bd-11e9-8600-ccefd73b8c9f.gif)
